shroudstone/replay.py

```diff
@@ -272,10 +273,26 @@ def process(self, chunk: pb.ReplayChunk):
         client_id = chunk.client_id
         timestamp = chunk.timestamp
 
-        if isinstance(msg, pb.Map):
-            self.map_name = msg.name
-            slot_count = player_slot_count[msg.name]
-            logger.debug(f"Setting up {slot_count} slots for map {msg.name}")
+        if isinstance(msg, pb.MapDetailsRecord):
+            self.map_name = msg.MapName
+            match(msg.MatchType):
```

Comment:
Please use if/else instead of match/case - this is only available in python 3.10+, would like to retain 3.8 compatibility unless there's a strong reason not to.

Reply:
Too much of a C boy my bad